GSA-TTS/FAC
# Change "Submission date" to "FAC Acceptance date"

# Change "Submission date" to "FAC Acceptance date"

Closes https://github.com/GSA-TTS/FAC/issues/3260

## Change:
1. Change the summary screen to display the FAC acceptance date rather than the date of submission. 

## How to Test:
1. Switch to this branch, run normally.
2. Navigate to a submissions summary page.
3. Verify that the new information is displayed, per the image below.

## Screenshot:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/deab7d0a-6d21-4e70-8019-a2e3e391aaef)


## PR checklist: submitters

- []   Make sure that whatever feature you’re adding has tests that cover the feature. This includes test coverage to make sure that the previous workflow still works, if applicable.

## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
